internal/tiltfile/docker_compose.go

```diff
@@ -293,6 +282,56 @@ func parseDCConfig(ctx context.Context, dcc dockercompose.DockerComposeClient, c
 	return services, nil
 }
 
+func getConfigAndServiceNames(ctx context.Context, dcc dockercompose.DockerComposeClient,
+	configPath string) (conf dcConfig, svcNames []string, err error) {
+	// calls to `docker-compose config` take a bit, and we need two,
+	// so do them in parallel to make things faster
+	configGot := make(chan error, 1)
+	servicesGot := make(chan error, 1)
+
+	go func() {
```

Comment:
the idiomatic way to do this is with an errgroup
https://godoc.org/golang.org/x/sync/errgroup#ex-Group--Parallel

Reply:
oh yeah, right! will do